libraries/core-react/src/components/DatePicker/PopupHeader.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react'
+import styled from 'styled-components'
+import { getMonth, getYear } from 'date-fns'
+import { datePicker as tokens } from './DatePicker.tokens'
+import { Icon, Typography } from '../../'
+import { arrow_back, arrow_forward } from '@equinor/eds-icons'
+import { typographyTemplate } from '../../utils'
+
+interface PopupHeaderProps {
```

Comment:
Can these props be taken the `react-datepicker` or `@types/react-datepicker`?

Reply:
couldnt find it in the first place, need to have another look, maybe together 👀